OpenKh.Tools.ModsManager/Services/ConfigurationService.cs

```diff
@@ -51,8 +51,9 @@ public static Config Open(string fileName)
             }
         }
 
-        private const string ConfigPath = "mods-manager.yml";
-        private const string EnabledModsPath = "mods.txt";
+        private static string _assString = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
Could you rename it to something more meaningful? Like `AppDirectory`, `AppPath` or similar

Reply:
:(